lib/MusicBrainz/Server/Report/WorkReport.pm

```diff
@@ -13,6 +15,8 @@ around inflate_rows => sub {
         map { $_->{work_id} } @$items
     );
 
+    $self->_load_extra_work_info(values %$works);
```

Comment:
Is this useful to have for a `{}` function? Is it just for future usage?

Reply:
It's used currently by `MusicBrainz::Server::Report::ISWCsWithManyWorks`.

Reply:
Oh, sure, it's for around or whatnot. Sorry, my bad.